### PR TITLE
Add modal popup to website

### PR DESCRIPTION
**Pull request by admirerbrown**
I did the following changes to my portfolio as instructed by the requirement:

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- You need to use a JavaScript array to store all of the information for all projects.
- You should implement popups for both mobile and desktop screen sizes
- added js linter config 
- implemented code review suggestions

### CODE REVIEW

File: modal_popup.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            <div class="modal-body">
+              <div class="modal-content">
+                <div class="modal-header">
+                  <h2 class="modal-title">${modalPopup.title[1]}</h2>
+                  <div class="tech-badges">
+                    <ul class="list p-lang mod-badge">
+                    ${modalPopup.language
+    .map(
+      (technology) => `
+                    <li class="btn-btn"> ${technology}</li>
```
Comment (mckent05) [inline modal_popup.js:114]: - This part of your project has some little difference from what is in the Figma design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
stjsdykxr

Design Template:
tjetet

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @admirerbrown,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Popup works as expected 👏 👏 
- PR summary and title are descriptive 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modal_popup.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
+// const viewProject = document.querySelectorAll(".btn").length;
```
Comment (anagudelogu) [inline modal_popup.js]: - [ ] **[OPTIONAL]:** Remember to delete all commented code! It makes your codebase look ugly and could potentially affect the readability and maintainability of the code in the future! I would suggest you fix this. 🤓 🛠️

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @admirerbrown, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**